### PR TITLE
[Notifier] [Slack] Include additional errors to slack notifier error message

### PR DESCRIPTION
**Pull request by norkunas**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

Sometimes it is hard to debug why notification was rejected, so I am proposing to include error descriptions to the exception message.

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Slack/Tests/SlackTransportTest.php [hunk 1]
```diff
@@ -239,4 +239,32 @@ public function testSendIncludesContentTypeWithCharset()
...
+        $response->expects($this->exactly(2))
+            ->method('getStatusCode')
+            ->willReturn(200);
+
+        $response->expects($this->once())
+            ->method('getContent')
+            ->willReturn(json_encode([
+                'ok' => false,
+                'error' => 'invalid_blocks',
+                'errors' => ['no more than 50 items allowed [json-pointer:/blocks]'],
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Slack/Tests/SlackTransportTest.php:256]: Are they talking about our option blocks? Should we also check the count of 50 beforehand to provide a better DX?

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/Notifier/Bridge/Slack/Tests/SlackTransportTest.php (hunk 1 above)
Comment (norkunas) [reply src/Symfony/Component/Notifier/Bridge/Slack/Tests/SlackTransportTest.php:256]: Its just one of many possible errors.. I could make a follow up PR for block limiting

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/Slack/Tests/SlackTransportTest.php:256]: Thanks I would appreciate sich PRs if we can give more infos beforehand to the developer 👍🏻

**Review by jderusse — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks for your work on this new feature!

**Review by Khnsimeon29 — APPROVED** [review]
Hello

**Pull request merged**